htaccess-2.4.dist

```diff
@@ -0,0 +1,28 @@
+# Apache config for version 2.4
+
+<IfModule mod_rewrite.c>
+    RewriteEngine on
+    RewriteRule ^(\.well\-known.*)$ /index.php/$1 [L]
+    RewriteRule ^\..* - [F]
+    RewriteCond %{REQUEST_FILENAME} !-f
+    RewriteCond %{REQUEST_FILENAME} !-d
+    RewriteCond $1 !^(index\.php)
+    RewriteRule ^(.*)$ /index.php/$1 [L]
```

Comment:
This doesn't work on my 2.4 setup unless I add a "?" after index.php on line 10.